docs/cudf/source/cudf_pandas/usage.md

```diff
@@ -1,10 +1,16 @@
 # Usage
 
-## Jupyter Notebooks and IPython
+There are three ways to enable `cudf.pandas`:
+1. In Jupyter notebooks
```

Comment:
Slight rephrasing to make it clear that the magics are not the only way to use cudf.pandas in notebooks. I'm not suggesting that we recommend option 3 in notebooks, I just want to avoid someone getting confused and asking "but can I do cudf.pandas.install() in notebooks too?"
```suggestion
1. IPython/Jupyter magics
```